### PR TITLE
picker update

### PR DESCRIPTION
**Pull request by w01fgang**

### CODE REVIEW

**Comment by 0455482** [comment]
1. In Calendar.js class in the selectDate function I added if / else statement to check if the returnState is false, and if its false i call the onSelect function that changes the returnState variable so i can select the dateTo without switching tab. After that when we select the second date, it goes in the else section where i call the function switchTab to switch the tab to the time picker, and the function onSelect to change the returnState to false (dateFrom) so i after that i can pick time for dateFrom.

2. In the DateTimePicker class in the function handleConfirmClick I removed the seting of the tab to 0, so i can add the onSelect function to change the returnState when picking time. After that i am checking if the returnState is true (dateTo) if thats true I'm calling the function onClose to close the modal. Also i binded onSelect function with the Calendar component.

**Comment by w01fgang** [comment]
Sorry for the delay.
I still see in the example single month selector, I expected to see similar to this:
expected

File: example/src/App.js [hunk 1]
```diff
@@ -1,113 +1,206 @@
...
-    } else {
-      this.setState({
-        date,
-        returnDate: date,
-      });
-    }
+    this.setDate = this.setDate.bind(this)
+    this.togglePicker = this.togglePicker.bind(this)
+    this.changeDateStatus = this.changeDateStatus.bind(this)
+    this.changeTimeStatus = this.changeTimeStatus.bind(this)
```
Comment (w01fgang) [inline example/src/App.js]: you don't need to bind methods as they already bound by `handleSelect = () => {}`

File: example/src/App.js [hunk 2]
```diff
@@ -1,113 +1,206 @@
...
     }
   }
 
-  setReturnDate = (date) => {
-    this.setState({
-      returnDate: date,
-    });
+  changeTimeStatus = () => {
+    console.log("change time state")
+    this.setState({timeState: 'timeTo', dateState: 'dateFrom'})
```
Comment (w01fgang) [inline example/src/App.js]: where do you use `timeState`?

File: example/src/App.js [hunk 3]
```diff
@@ -1,113 +1,206 @@
...
+          value={this.state.fromDate}
+          returnValue={this.state.toDate}
+          onSelect={this.changeDateStatus}
           onChange={this.setDate}
-          onSelect={this.handleSelect}
-          onClose={this.closeDatePicker}
+          onClose={this.togglePicker}
           language="en"
-          label={returnState ? 'Return' : 'Get'}
+          label="Some Label"
```
Comment (w01fgang) [inline example/src/App.js]: The label explains which date the user is currently selecting.

**Pull request merged**